l5kit/l5kit/simulation/unroll.py

```diff
@@ -0,0 +1,226 @@
+from typing import Dict, List, NamedTuple, Optional, Tuple
+
+import numpy as np
+import torch
+from torch.utils.data.dataloader import default_collate
+from tqdm.auto import tqdm
+
+from l5kit.data import AGENT_DTYPE
+from l5kit.dataset import EgoDataset
+from l5kit.geometry import transform_points
+from l5kit.simulation.dataset import SimulationDataset
+
+
+class SimulationConfig(NamedTuple):
+    """ Defines the parameters used for the simulation of ego and agents around it.
+    :param use_ego_gt: whether to use GT annotations for ego instead of model's outputs
```

Comment:
```suggestion
    """ Defines the parameters used for the simulation of ego and agents around it.
    
    :param use_ego_gt: whether to use GT annotations for ego instead of model's outputs
```